TGC/localisation/00_events_1000000000.csv

```diff
@@ -1,14 +1,14 @@
 CODE;ENGLISH;FRENCH;GERMAN;POLISH;SPANISH;ITALIAN;SWEDISH;CZECH;HUNGARIAN;DUTCH;PORTUGUESE;RUSSIAN;FINNISH;x
-EVTDESC1904191417;The call has come for Tabasco to join Yucatán, a union of two brother-states which will tend to the needs of their people.;;;;;;;;;;;;;x
+EVTDESC1904191417;The call has come for Tabasco to join Yucat�n, a union of two brother-states which will tend to the needs of their people.;;;;;;;;;;;;;x
```

Comment:
Please confirm this file doesn't have the wrong encoding, looking at the changes it seems like it might.

Reply:
It marks all special characters as changed because I've changed them all: it was (and still is you can check the file on the master branch) all encoded in UTF-8

Reply:
Then the file was wrong to begin with, it has to be encoded in Win1252. The game will not parse a UTF8 file correctly.